apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/MapLikeToContainInAnyOrderCreatorSamples.kt

```diff
@@ -0,0 +1,52 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.expect
+import kotlin.test.Test
+
+class MapLikeToContainInAnyOrderCreatorSamples {
+
+    @Test
+    fun entry() {
+        expect(mapOf(1 to "a", 2 to "b")) toContain o inAny order entry (2 to "b")
+
+        fails {
+            expect(mapOf(1 to "a", 2 to "b")) toContain o inAny order entry (2 to "c")
+        }
+    }
+
+    @Test
+    fun entryKeyValue() {
+        expect(mapOf(1 to "apple", 2 to "banana")) toContain o inAny order entry (
+            keyValue(2) { this toStartWith "b" }
+        )
+
+        fails {
+            expect(mapOf(1 to "apple", 2 to "banana")) toContain o inAny order entry (
+                keyValue(1) { this toStartWith "b" } // fails because subject does not have this entry
```

Comment:
Based on the feedback from #1565, I will update the comments to be more explanatory

Reply:
Good, waiting for the adjustments and will merge afterwards